quantum-adapters/kiwoom-adapter/src/kiwoom_api/analysis/indicators/dart_integrated_indicators.py

```
@@ -0,0 +1,375 @@
+            "D38": dart_disclosure.get("has_earnings_surprise", False),  # 호재 임박
+            "D39": False,  # 주도 테마 (수동 입력 필요)
+            "D40": integrated_data.get("institutional_holding_change", 0),  # 기관 수급
+            "D41": dart_financial.get("dividend_yield", 0),  # 배당수익률 (DART)
+            "D42": dart_disclosure.get("has_earnings_surprise", False),  # 어닝서프라이즈
+            "D43": dart_disclosure.get("unfaithful_disclosures", 0) > 0,  # 불성실 공시
+            "D44": dart_disclosure.get("negative_disclosures", 0) > 2,  # 악재뉴스
+            "D45": dart_disclosure.get("positive_disclosures", 0) > 5,  # 호재뉴스 도배
+            "D46": dart_financial.get("interest_coverage_ratio", 999),  # 이자보상배율 (DART)
+            
+            # 메타 정보
+            "stock_code": integrated_data.get("stock_code", stock_code),
```

Ruff (0.12.2)

Undefined name `stock_code`

(F821)